### ISSUE TITLE
Difference between Category and Categories layouts

### ISSUE THREAD

**Issue by Funeoz** [mmistakes/jekyll-theme-basically-basic#84]
What's the difference between Category and Categories because I'm confused ?

**Comment by mmistakes** [comment]
I thought the documentation explained this clearly, but here's another take.

## Categories layout

Shows all categories currently used on your site, with the posts attributed to them. For example if you have two categories `foo` and `bar` you'd get a page like:

```
## foo
- Some post that is in the foo category
- Another post that is in the foo category

## bar
- Some other post that is in the bar category
```

The category layout is for building a page that only lists 1 category and all the posts in it. So if you wanted an archive page for just the `foo` posts you'd create a "category" page for it.

This page would look something like this:

```
## foo
- Some post that is in the foo category
- Another post that is in the foo category
```

To summarize, categories layout shows all categories and their posts, category is for a single category and its post. The same is true for the `tag` and `tags` layouts.

**Comment by mmistakes** [comment]
Visuals may help...

this is a [`layout: categories`](https://mmistakes.github.io/jekyll-theme-basically-basic/categories/) page (shows all categories and their posts)
and this is a [`layout: category`](https://mmistakes.github.io/jekyll-theme-basically-basic/categories/edge-case/) page (shows just the "edge case" category and its posts)

**Comment by Funeoz** [comment]
So, how can I see a category link on the homepage of my site ?
